scripts/create_clone

```diff
@@ -51,20 +62,28 @@ def parse_command_line(args):
                "Must specify -clone as an input argument")
 
     return args.case, args.clone, args.keepexe, args.mach_dir, args.project, \
-        args.cime_output_root
+        args.cime_output_root, args.user_mods_dir
 
 ##############################################################################
 def _main_func():
 ###############################################################################
 
-    case, clone, keepexe, mach_dir, project, cime_output_root = parse_command_line(sys.argv)
+    case, clone, keepexe, mach_dir, project, cime_output_root, user_mods_dir = parse_command_line(sys.argv)
 
     cloneroot = os.path.abspath(clone)
     expect(os.path.isdir(cloneroot),
            "Missing cloneroot directory %s " % cloneroot)
 
+    if user_mods_dir is not None:
+        if os.path.isdir(user_mods_dir):
+            user_mods_dir = os.path.abspath(user_mods_dir)
+
     with Case(cloneroot, read_only=False) as clone:
-        clone.create_clone(case, keepexe, mach_dir, project, cime_output_root)
+        if user_mods_dir is not None:
```

Comment:
Don't need this if else block.   clone.create_clone(case, keepexe, mach_dir, project, cime_output_root, user_mods_dir) will work in both cases.

Reply:
Good point.